.travis.yml

```diff
@@ -17,8 +17,7 @@ rvm:
   - 2.6.0
   - 2.5.3
   - 2.4.5
-  - 2.3.8
-  - jruby-9.1.12.0
+  - jruby-head
```

Comment:
Why `jruby-head`?

Reply:
I think it's the most useful option to have on CI because it will update automatically.

Reply:
We run specs against released versions of `jruby` so it would be better to use `9.2.7.0` here (although it's kinda borked on travis now 😓 )